### PR TITLE
Clone upScriBt to the ScriBt folder

### PR DESCRIPTION
**Pull request by timschumi**
Clone upScriBt to the ScriBt folder, not the current working directory

### CODE REVIEW

File: ROM.sh [hunk 1]
```diff
@@ -1440,7 +1440,7 @@ function the_start() # 0
 
     if [[ "$BRANCH" =~ (master|staging) ]]; then
         # Download the Remote Version of Updater, determine the Internet Connectivity by working of this command
-        curl -fs -o upScriBt.sh https://raw.githubusercontent.com/a7r3/ScriBt/${BRANCH}/upScriBt.sh && \
+        curl -fs -o ${PATHDIR}upScriBt.sh https://raw.githubusercontent.com/a7r3/ScriBt/${BRANCH}/upScriBt.sh && \
```
Comment (a7r3) [inline ROM.sh:1443]: @TimSchumi [Line 1436](https://github.com/a7r3/ScriBt/blob/master/ROM.sh#L1436) takes care of it

Comment (a7r3) [reply ROM.sh:1443]: Oh, wait, keep it.

Else move line 1439 (`cd` back to current dir) just after the `if` condition

Comment (timschumi) [reply ROM.sh:1443]: Line 1436 seems not to work. In my End it still curls it to my current dir.
This change fixed it

Comment (a7r3) [reply ROM.sh:1443]: Cool. And better that [Line 1436](https://github.com/a7r3/ScriBt/blob/master/ROM.sh#L1436) is present in [upScriBt](https://github.com/a7r3/ScriBt/blob/master/upScriBt.sh#L6) itself, I'm fine with this

**Comment by timschumi** [comment]
Ready to merge (at least from my side)

**Pull request merged**